jestSetup.js

```diff
@@ -1,2 +1,2 @@
-// eslint-disable-next-line n/no-unpublished-require
+ 
```

Comment:
This is happening when I run `npm i` in the package 😕

Reply:
Does it cause a lint error? If not, it's fine that it's removed.

Reply:
No more lint errors than are currently in main :D